dvc/config.py

```diff
@@ -2,16 +2,20 @@
 
 from __future__ import unicode_literals
 
-from dvc.utils.compat import str, open
-
 import os
 import re
 import copy
 import errno
+import logging
```

Comment:
The following `configobj` is not a part of the standard lib, so we need to separate it from the upper standard imports with a newline.
```suggestion
import logging

```

Reply:
@sofia100 Still missing a newline here :)